### PR TITLE
Add Actions Docs

### PR DESCRIPTION
**Pull request by RoseBlume**
#### Description

- What does this PR change? Give us a brief description. 
- Closes #

### CODE REVIEW

**Comment by RoseBlume** [comment]
@FabianLars This replaces the previous one. I couldn't fix the errors so I tried from scratch.

File: src/content/docs/develop/actions.mdx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+      - release
+
+jobs:
+  publish-tauri:
+    permissions:
+      contents: write
+    strategy:
+      fail-fast: false
+      matrix:
+        settings:
```
Comment (brenoepics) [inline src/content/docs/develop/actions.mdx]: Can you change this `settings` to `include`?
reference: https://github.com/tauri-apps/tauri-action/commit/e5c4e13a7c979603440eddb183b266170613b1ba / https://github.com/tauri-apps/tauri-action/pull/772

It appears that `settings` should not be used this way according to the GitHub actions documentation and intellisense.
image

File: src/content/docs/develop/actions.mdx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+          - platform: 'macos-latest' # for Arm based macs (M1 and above).
+            args: '--target aarch64-apple-darwin'
+          - platform: 'macos-latest' # for Intel based macs.
+            args: '--target x86_64-apple-darwin'
+          - platform: 'ubuntu-22.04' # for Tauri v1 you could replace this with ubuntu-20.04.
+            args: ''
+          - platform: 'windows-latest'
+            args: ''
+
+    runs-on: ${{ matrix.settings.platform }}
```
Comment (brenoepics) [inline src/content/docs/develop/actions.mdx]: needs to change it to `runs-on: ${{ matrix.platform }}`

File: src/content/docs/develop/actions.mdx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: install dependencies (ubuntu only)
+        if: matrix.settings.platform == 'ubuntu-22.04' # This must match the platform value defined above.
```
Comment (brenoepics) [inline src/content/docs/develop/actions.mdx]: needs to change it to `if: matrix.platform == 'ubuntu-22.04'`

File: src/content/docs/develop/actions.mdx [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+        uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+          cache: 'yarn' # Set this to npm, yarn or pnpm.
+
+      - name: install Rust stable
+        uses: dtolnay/rust-toolchain@stable # Set this to dtolnay/rust-toolchain@nightly
+        with:
+          # Those targets are only used on macos runners so it's in an `if` to slightly speed up windows and linux builds.
+          targets: ${{ matrix.settings.platform == 'macos-latest' && 'aarch64-apple-darwin,x86_64-apple-darwin' || '' }}
```
Comment (brenoepics) [inline src/content/docs/develop/actions.mdx]: same here

**Review by brenoepics — CHANGES_REQUESTED** [review]

**Comment by brenoepics** [comment]
Raspberry Pi one is fine, good job btw

File: src/content/docs/develop/actions.mdx (hunk 1 above)
Comment (RoseBlume) [reply src/content/docs/develop/actions.mdx]: Works fine on my repo. It doesnt make a difference to me so its fine if you prefer include.

Comment (RoseBlume) [reply src/content/docs/develop/actions.mdx]: I'll change it to match the proper usage case.

**Review by FabianLars — APPROVED** [review]
sorry for hijacking your PR 😅

**Pull request merged**